skills_DS/skills_DS/settings.py

```diff
@@ -163,7 +163,7 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 EMAIL_HOST_USER = 'cosc448atesting@gmail.com'
-EMAIL_HOST_PASSWORD = ''
+EMAIL_HOST_PASSWORD = 'xEpdu0-waqqux-repwuf'
```

Comment:
Time to change the password again xD

I just notice we never merged the change for the environment variable change. I will open a pull request on it later tomorrow.

Reply:
I'll hide it again lol